src/schnorrsig.rs

```diff
@@ -0,0 +1,471 @@
+//! # Schnorrsig
+//! Support for Schnorr signatures.
+//!
+
+#[cfg(any(test, feature = "rand"))]
+use rand::Rng;
+
+use super::Error::{InvalidSchnorrPublicKey, InvalidSchnorrSignature, InvalidSecretKey};
+use super::{from_hex, Error};
+use core::{fmt, str};
+use ffi::{self, CPtr};
+use {constants, Secp256k1, SecretKey};
+use {Message, Signing};
+
+/// Represents a schnorr signature.
+pub struct SchnorrSignature([u8; constants::SCHNORR_SIGNATURE_SIZE]);
```

Comment:
Similarly we can drop the prefix from this type

Reply:
Done